src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Take a look at the task's [checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md), number 2

Reply:
done)